src/main/resources/static/js/webauthn.js

```diff
@@ -0,0 +1,145 @@
+function registerWebAuthnCredentials(widgetSessionId) {
```

Comment:
Shall we put a divider commment between registration (app simulator) and authentication (widget)?

Reply:
+1

Reply:
+1

Reply:
I'm totally not a friend of divider comments. Reasons: 
- You always forget them when refactoring / moving code, so they lose their meaning and are more confusing than helpful pretty soon.
- There are functions in this file which are used by both, registration and authentication. Another separator for that?

Reply:
If you wanna separate it, I could create multiple files. 
(Totally out of scope: Once we have a build pipeline for our widget, we would split up those files anyways and have them re-combined during the build)

Reply:
Got your point now. 😉 I will add comments to the methods only used for the app simulation.